### DISCUSSION TITLE
Cannot get website to generate meetings anymore with current theme[Bug]:

### DISCUSSION THREAD

**Discussion by cm359531** [code4recovery/12-step-meeting-list#643] [General]
### Website With Issue

http://barringtonaa.org/meetings/

### What happened?

All other websites or links attached it to no longer will open or work. Need to get this fixed asap please

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Firefox, Chrome, Microsoft Edge

**Comment by 97eighty** [comment]
@cm359531 Hi, Which theme we’re using when it worked and which theme are you using now? Also, I’m not understanding what you mean by “not generating meetings”. Is this an export?

**Comment by cm359531** [comment]
I'm using the theme oceanWP. I added the new version of this to the website and it looks like it's no longer showing the meeting page correctly. 

There is no filter and the individual pages that linked to the that page are no hope or it's unavailable?

**Comment by 97eighty** [comment]
@cm359531 The page linked above shows for me correctly. Try clearing your local cache or taking a look in Incognito or Private mode.

**Comment by brianw-area24** [comment]
@cm359531 Just following up on this. I tried the link a couple days ago, and again this morning, and both times I'm seeing meetings as expected. However, if you're still having issues please let us know and we'll look into it farther.
